test/easyconfigs/suite.py

```diff
@@ -45,6 +45,9 @@
 fancylogger.disableDefaultHandlers()
 fancylogger.setLogLevelError()
 
+# make sure no deprecated behaviour is triggered
```

Comment:
yes, the tests should never trigger deprecated (or no longer supported) behaviour...